app/views/comments/edit.html.erb

```diff
@@ -8,7 +8,7 @@
       <div class='col-sm-10 col-sm-offset-1'>
         <%= render partial: 'comments/header' %>
 
-        <%= render partial: 'comments/form', locals: { url: comment_path(@comment), cancel_path: after_update_object_path_for(@comment.commentable), submit_text: "Update" } %>
+        <%= render partial: 'comments/form', locals: { url: comment_path(@comment), cancel_path: after_update_object_path_for(@comment.commentable), submit_text: "#{t('comments.edit.submit')}" } %>
```

Comment:
Seems weird to have a value of "Update" for a property called `comments.edit.submit`.  Maybe that's better named as `comments.edit.update`?